src/fielddef.ts

```diff
@@ -151,7 +151,7 @@ export interface FieldDefBase<F> {
 export interface FieldDef<F> extends FieldDefBase<F> {
   /**
    * The encoded field's type of measurement (`"quantitative"`, `"temporal"`, `"ordinal"`, or `"nominal"`).
-   * It can also be a geo type (`"latitude"`, `"longitude"`, and `"geojson"`) when projecting geographical coordinates using a `"point"` mark (on `"x"`, and `"y"` channels), a `"rule"` mark (on `"x"`, `"x2"`, `"y"`, and `"y2"` channels) or projecting a GeoJSON shapefile on a `"geoshape"` mark (on the `"shape"` channel).
+   * It can also be a geo type (`"latitude"`, `"longitude"`, and `"geojson"`) when when a [geographic projection](projection.html) is applied.
```

Comment:
when when

Reply:
https://media.giphy.com/media/C87IXdLfJ44Zq/giphy.gif

Reply:
I still see it in the schema